docs/antctl.md

```diff
@@ -181,12 +185,25 @@ antrea-network-policy.md#antrea-native-policy-ordering-based-on-priorities).
 antctl get networkpolicy --sort-by=effectivePriority
 ```
 
-Antrea Agent additionally supports printing NetworkPolicies applied to a
-specified local Pod using this `antctl` command:
+Antrea Agent supports some extra `antctl` commands.
 
-```bash
-antctl get networkpolicy -p pod -n namespace
-```
+* Printing NetworkPolicies applied to a specific local Pod.
+
+  ```bash
+  antctl get networkpolicy -p pod -n namespace
```

Comment:
```suggestion
  antctl get networkpolicy -p POD -n NAMESPACE
```
variables in example are normally in uppercase to be different from the commands and options

Reply:
Got it.